Sources/KituraNet/HTTP/HTTPServer.swift

```diff
@@ -347,7 +343,7 @@ public class HTTPServer: Server {
         }
 
         let bootstrap = ServerBootstrap(group: eventLoopGroup)
-            .serverChannelOption(ChannelOptions.backlog, value: BacklogOption.Value(self.maxPendingConnections))
+            .serverChannelOption(ChannelOptions.backlog, value: ChannelOptions.Types.BacklogOption.Value(self.maxPendingConnections))
```

Comment:
```suggestion
            .serverChannelOption(ChannelOptions.backlog, value: self.maxPendingConnections)
```

Reply:
Unfortunately `maxPendingConnections` is an Int and "value" expects a type of `ChannelOptions.Types.BacklogOption.Value` so I cannot make this change.

Reply:
ah, okay, you could write `CInt(self.maxPendingConnections)` or `Int32(self.maxPendingConnections)` or just `.init(self.maxPendingConnections)` if you wanted but I get your point

Reply:
Oh I see.  I like the `.init()` approach.  Thanks for the tip!